contrib/exporters/core/extend.go

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package core
+
+import (
+	"fmt"
+	"time"
+	"encoding/json"
+	"github.com/skydive-project/skydive/api/client"
+	"github.com/skydive-project/skydive/logging"
+	"github.com/spf13/viper"
+	"strings"
+	cache "github.com/pmylund/go-cache"
+)
+
+// extendNone is used for no transform extensions
+type extendNone struct {
+}
+
+func NewExtendNone(cfg *viper.Viper) (interface{}, error) {
+	return &extendNone{}, nil
+}
+
+func (e *extendNone) Extend(in interface{}) interface{} {
+	return in
+}
+
+// extendGremlin is used for transform extensions using gremlin expressions
+// Syntax supported is:
+//	VAR_NAME=<gremlin expression with a single substitution string>
```

Comment:
wrong

Reply:
This is an old version of the code. Not sure why the new code doesn't show up here.

Reply:
This file (core/extend.go) should have been removed. Please ignore code in this file.